src/assets/stylesheets/style.css

```diff
@@ -0,0 +1,276 @@
+/*** Project Meta ***/
+
+/* general - line 13 */
+
+/* header - line 39 */
+
+/* main content - line 19 */
+
+/* footer - line 144 */
+
+/** GENERAL **/
+
+/* @font-face {
+  font-family: 'MyFont';
+  src: url('./my-font.woff2') format('woff2'),
+    url('./my-font.woff') format('woff');
+  font-weight: 600;
+  font-style: normal;
+} */
+
+/* :root {
+  --border-bottom: 1px solid rgb(230, 230, 230);
+  --grey: rgb(246, 246, 246);
+  --flex: flex;
+  --border-radius: 10px;
+} */
+
+/* Mobile View */
+
+.html {
```

Comment:
- [ ]  OPTIONAL:  I can see you did a mobile view for your app! Congratulations.  I recommend fixing the header, so all the elements are accessible, or even hide the ones that are not used:

![image](https://user-images.githubusercontent.com/57421823/164952751-2ae1117c-e88b-40e1-8c2e-25bf869f9782.png)

![image](https://user-images.githubusercontent.com/57421823/164952757-090e90ac-81fd-4986-a0e3-0c3a80ca2cf7.png)